mavros/src/plugins/hil_actuator_controls.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @brief HilActuatorControls plugin
+ * @file hil_actuator_controls.cpp
+ * @author Pavel Vechersky <pvechersky@student.ethz.ch>
+ * @author Beat Küng <beat-kueng@gmx.net>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2016 Pavel Vechersky & Beat Küng.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+
+#include <mavros_msgs/HilActuatorControls.h>
+
+namespace mavros {
+namespace std_plugins {
+/**
+ * @brief Hil Actuator Control plugin
+ */
+class HilActuatorControlsPlugin : public plugin::PluginBase {
+public:
+	HilActuatorControlsPlugin() : PluginBase(),
+		hil_actuator_controls_nh("~hil_actuator_controls")
+	{ }
+
+	void initialize(UAS &uas_)
+	{
+		PluginBase::initialize(uas_);
+
+		hil_actuator_controls_pub = hil_actuator_controls_nh.advertise<mavros_msgs::HilActuatorControls>("hil_actuator_controls", 10);
```

Comment:
Do you really need that long topic name: */mavros***/hil_actuator_controls/hil_actuator_controls**

I thing you may use `~` in node handle init.